R/tm_g_patient_profile.R

```diff
@@ -794,9 +798,11 @@ srv_g_patient_profile <- function(id,
       teal.code::chunks_safe_eval()
 
       if (select_plot["rs"]) {
-        validate(
-          need(!is.null(rs_var), "Please select a tumor response variable.")
-        )
+        iv_tum <- shinyvalidate::InputValidator$new()
+        iv_tum$add_rule("rs_var", shinyvalidate::sv_required())
+        iv_tum$enable()
+        validate(need(iv_tum$is_valid(), "Misspecification error: please observe red flags in the interface."))
```

Comment:
this validation and the following ones below can be added in the  very first `shinyvalidate` section to avoid repeated sections.